sched/sched/sched_smp.c

```diff
@@ -314,6 +318,17 @@ int nxsched_smp_call_async(cpu_set_t cpuset,
       sched_lock();
     }
 
+  if (CPU_ISSET(this_cpu(), &cpuset))
+    {
+      ret = data->func(data->arg);
+      if (ret < 0)
+        {
+          goto out;
+        }
+
+      CPU_CLR(this_cpu(), &cpuset);
```

Comment:
let's call nxsem_post here too to remove the special process in nxsched_smp_call

Reply:
ok